client/dist/styles.css

```diff
@@ -1019,23 +1024,35 @@ button.sarah-right-arrow:hover {
   margin-bottom: 1%;
   display: flex;
   justify-content: flex-start;
+padding-bottom: 1%;
 }
 
 .qa_question {
-  background-color: antiquewhite;
+  background-color: #DED1BD;
   font-weight: bold;
   /* text-decoration: underline; */
   margin-bottom: 0.5%;
+  display: initial;
+
 }
 
 
 .qa_answerList {
-  border-left: 1px solid black;
-  padding-left: 0.8%
+  border-left: 0.5rem solid #D49E8D;
+  padding-left: 0.8%;
+  background-color: white;
+  border: 2px solid #B08401;
+  box-shadow: rgba(100, 100, 111, 0.3) 0px 7px 29px 0px;
+  border-radius: 6px;
+  margin-bottom: 1%;
+  padding-top: 1%;
+  margin-right: 2%;
+  padding-left: 1%;
```

Comment:
It looks like you have duplicates of the padding-left property -- one with .8% and another with 1%.

Might be helpful to refactor this to use the Xpx Xpx Xpx Xpx format for border, padding, and margin to clean up this code a smidge.

Reply:
Thanks, I will change the % to px.